### PR TITLE
Patch the editUrl for the docs

### PR DESCRIPTION
**Pull request by bdfinst**
## Description

There’s a weird error in the docusaurus content-docs plugin. It’s dropping the last segment of the editUrl path
This editUrl: 'https://github.com/defenseunicorns/zarf/tree/main/x/' results in https://github.com/defenseunicorns/zarf/tree/main/ as the base path. For now, I’ll apply the hacky dummy x path. I may look at their code later to see why their plugin is broken.

### CODE REVIEW

File: docs-website/package.json [hunk 1]
```diff
@@ -14,9 +14,9 @@
     "write-heading-ids": "docusaurus write-heading-ids"
   },
   "dependencies": {
-    "@docusaurus/core": "2.3.0",
-    "@docusaurus/preset-classic": "2.3.0",
-    "@docusaurus/theme-mermaid": "^2.3.0",
+    "@docusaurus/core": "^2.3.1",
```
Comment (jeff-mccoy) [inline docs-website/package.json:17]: we can bump deps, but renovate will pin them again so can we drop the `^`?

**Review by jeff-mccoy — COMMENTED** [review]
quick comment, but tested it on a few pages and it does seem to be working

**Comment by Racer159** [comment]
> 

same, and agree with that comment @jeff-mccoy

File: docs-website/package.json (hunk 1 above)
Comment (bdfinst) [reply docs-website/package.json]: Yep. Didn't notice it added them back. Fixed.

**Review by jeff-mccoy — APPROVED** [review]

**Pull request merged**